places-rest-apis/adobe-i-o-integration/README.md

```diff
@@ -66,23 +65,35 @@ The following video guides you through the process of generating the key pair:
 To create a Places integration:
 
 1. Go to [https://console.adobe.io/](https://console.adobe.io/) and sign in with your Adobe ID.
-2. If you have access to more than one Experience Cloud organization, select the organization from the drop-down list on the left.
-3. Click **New Integration**.
-4. Select **Access an API** and click **Continue**.
-5. Under **Experience Cloud**, select **Places** as the Adobe service to which you want to integrate and click **Continue**.
+
+2. Click on the **Create integration** in the **Quick Start** section.
```

Comment:
Please reverse the order like this:
"2. In the **Quick Start** section, click **Create integration** .